src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+
+public class Storage {
+    public static final HashMap<Fruit, Integer> storageDate = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<Fruit, Integer> storageDate = new HashMap<>();
```
Use an interface instead its implementation